config/locales/fr.yml

```diff
@@ -158,6 +164,11 @@ fr:
           collaborative_drafts_list: Accéder aux brouillons collaboratifs
           new_proposal: Nouvelle proposition
           view_proposal: Voir la proposition
+      notifications:
```

Comment:
This locale key is no longer needed because it is called above with `notification_title` key